resources/js/components/dashboard/ControlGroup.vue

```diff
@@ -1,7 +1,19 @@
 <template>
-  <div class="control-group container">
-    <h1>Control Group</h1>
+<div class="control-group container">
+  <div class="row">
+    <div class="col-8 offset-2 text-center">
+      <div class="control-group__body">
+        <i class="control-group__icon fas fa-user" style="color:#299afb"></i>
```

Comment:
Consider adding the icon styling to the `control-group__icon` class instead of inline styling

Reply:
you right